contrib/plugins/cowsay.sh

```diff
@@ -0,0 +1,25 @@
+#! /bin/sh
+
+# Eg. {"jsonrpc":"2.0","id":2,"method":"getmanifest","params":{}}\n\n
+read JSON
+read BLANK
+id=$(echo "$JSON" | sed 's/.*"id" *: *\([0-9]*\),.*/\1/')
+
+echo '{"jsonrpc":"2.0","id":'$id',"result":{"options":[],"rpcmethods":[{"name":"cowsay","usage":"<string>","description":"Have a cow, man!"}]}}'
+
+# Eg. {"jsonrpc":"2.0","id":5,"method":"init","params":{"options":{},"configuration":{"lightning-dir":"/home/rusty/.lightning","rpc-file":"lightning-rpc","startup":false}}}\n\n
+read JSON
+read BLANK
+
+# eg. { "jsonrpc" : "2.0", "method" : "cowsay", "id" : 6, "params" :[ "hello"] }
+while read JSON; do
+    echo "$JSON" > /tmp/cowsay
+    read BLANK
+    id=$(echo "$JSON" | sed 's/.*"id" *: *\([0-9]*\),.*/\1/')
+    params=$(echo "$JSON" | sed 's/.*"params" *: *//' | tr -d '[{}]"')
+    echo '{"jsonrpc":"2.0","id":'$id',"result":{"format-hint":"simple","cowsay":"'
+    # FIXME: lightning-cli does not unescape \\, so we replace with an L.
+    printf "$params" | cowsay | sed 's/\\/L/g' | sed ':a;N;$!ba;s/\n/\\n/g' | tr '\012' '"'
```

Comment:
This depends on an external application which increases burden when packaging and distributing the plugins. I say we write a cowsay implementation in bash for maximum portability.

Reply:
```
 _________________________________________
/ Chose to add an emergency backup cow,   \
| because while we're only using a        |
| fraction of cowsay(1) powers now,       |
\ future generations may add more options /
 -----------------------------------------
        \   ^__^
         \  (**)\_______
            (__)\       )\/\
             U  ||----w |
                ||     ||
```

Reply:
I experimented in an other ln project with a patch where the cow says something with nc and ssh that can remotely mesmerize her miind, now lightning has finely thanks to the new high priest of the cult of dead cow **rusty** and his young knight **jb** this pedantic **swift** and the most cynic ai **Zmn** armitage.like magic powers. please ping me for the patch, as I think this is to much magic for one to not share it. :smile: